web-app/django/VIM/apps/instruments/views/instrument_detail.py

```diff
@@ -0,0 +1,45 @@
+from django.views.generic import DetailView
+from VIM.apps.instruments.models import Instrument, Language

```

Comment:
`Lanugage` import is not used...are you using pylint for linting?

Also, I see now that this doesn't have the right dependencies for typing -- I'll fix this.

Reply:
I realize my message wasn't clear...I'll fix the typing dependencies, but would you remove the `Language` import?

Reply:
> I realize my message wasn't clear...I'll fix the typing dependencies, but would you remove the `Language` import?

Thank you for the clarification. In my latest commit, I kept the `Language` model for future support English and France detail page. The usage is similar to instrument list page:

```
active_language_en = self.request.session.get("active_language_en", None)
context["active_language"] = (
    Language.objects.get(en_label=active_language_en)
    if active_language_en
    else Language.objects.get(en_label="english")  # default in English
)
```

If it is necessary to support two languages, I'll keep the `Language` import.

Reply:
Ah ok, I didn't see that it is now used.